### DISCUSSION TITLE
Problem installing the OR-Tools library for C++ on Windows 10

### DISCUSSION THREAD

**Discussion by ultramaga** [google/or-tools#4029] [Build and configuration issues]
What version of OR-Tools and what language are you using?
Version: v9.7 binary
Language: C++

What operating system (Linux, Windows, ...) and version?
Windows 10, Studio Community 2022 (64-bit) 

What did you do?

    Followed the following instructions:
    https://developers.google.com/optimization/install/cpp/binary_windows

What did you expect to see
Successful test

What did you see instead?
error LNK2038: mismatch detected for '...': value '0' doesn't match value '2' in basic_example.obj

Thanks for any help.

[or_tools_problem.txt](https://github.com/google/or-tools/files/13732443/or_tools_problem.txt)

**Comment by lperron** [comment]
Please specify release mode. (-DCMAKE_BUILD_TYPE=Release)

**Reply by lperron** [reply]
See https://github.com/google/or-tools/issues/3822

**Comment by ultramaga** [comment]
Thanks for your advice Laurent.

I added the parameter you suggested but it resulted in the same error.  I have added some more files that I hope can be useful in diagnosing my problem.

[cmake_results.txt](https://github.com/google/or-tools/files/13743642/cmake_results.txt)
[CTestTestfile.cmake.txt](https://github.com/google/or-tools/files/13743652/CTestTestfile.cmake.txt)
[cmake_install.cmake.txt](https://github.com/google/or-tools/files/13743653/cmake_install.cmake.txt)
[CMakeCache.txt](https://github.com/google/or-tools/files/13743654/CMakeCache.txt)
[CMakeError.log.txt](https://github.com/google/or-tools/files/13743655/CMakeError.log.txt)
[CMakeOutput.log.txt](https://github.com/google/or-tools/files/13743656/CMakeOutput.log.txt)
